source/views/layouts/govuk_template.html.erb

```diff
@@ -110,7 +110,12 @@
             <%= yield :footer_support_links %>
 
             <div class="open-government-licence">
-              <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence</a></p>
+              <% if content_for?(:licence_logo) %>
+                <%= yield :licence_logo %>
```

Comment:
@fofr I don't disagree with you, just not sure what the alternative would be within the constraints of the multiple templating languages we have to deal with?

Happy for this PR to be declined for now though until a better solution comes up...